forge-gui/res/cardsfolder/upcoming/bhaal_lord_of_murder.txt

```diff
@@ -0,0 +1,12 @@
+Name:Bhaal, Lord of Murder
+ManaCost:2 B R G
+Types:Legendary Creature God
+PT:4/4
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | CheckSVar$ X | SVarCompare$ LEY | Description$ As long as your life total is less than or equal to half your starting life total, CARDNAME has indestructible.
+SVar:X:Count$YourLifeTotal/Times.2
+SVar:Y:Count$YourStartingLife
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.nonToken+Other+YouCtrl | TriggerZones$ Battlefield | Execute$ DBPutCounter | TriggerDescription$ Whenever another nontoken creature you control dies, put a +1/+1 counter on target creature and goad it.
+SVar:DBPutCounter:DB$ PutCounter | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 1 | RememberCards$ True | SubAbility$ DBGoad
+SVar:DBGoad:DB$ Goad | Defined$ Remembered | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```

Comment:
I think for line 11 you can use `Defined$ Targeted`

Then you won't need RememberCards, etc.